provisioning/device/samples/Getting Started/X509Sample/Parameters.cs

```diff
@@ -41,6 +41,7 @@ internal class Parameters
         public string CertificatePassword { get; set; }
 
         [Option(
+            'g',
```

Comment:
Why give something that won't likely be changed a shortcut?

Our take has generally evolved to only having aliases for the required/commonly changed params.